scripts/xtensa-build-zephyr.py

```diff
@@ -470,7 +486,13 @@ def build_platforms():
 			build_cmd.append(f"-DOVERLAY_CONFIG={overlays}")
 
 		# Build
-		execute_command(build_cmd, cwd=west_top)
+		try:
+			execute_command(build_cmd, cwd=west_top)
+		except:
+			zephyr_path = pathlib.Path(west_top, "zephyr")
+			if not os.path.exists(zephyr_path):
+				print("Zephyr project not found. Please run this script with -c flag or clone manually.")
+				exit(-1)
```

Comment:
sys.exit("Zephyr project not found. Please run this script with -c flag or clone manually."), see above (or below?) why.